ISSUE_TEMPLATE.md

```diff
@@ -1,36 +1,62 @@
+----------------------------- Delete below -----------------------------
 
-Please fill the info fields, it helps to get you faster support ;)
+INSTRUCTIONS
+============
+If you do not follow these instructions, your issue may be dismissed.
+
+1. Follow the checklist and fill in the [ ] spaces with an X.
+2. Fill in all the fields marked with [ ] (pick the correct option for you in each case, delete the others).
+3. If you haven't already done so, test your issue against current master branch (aka latest git), because it may have been already fixed.
+4. Describe your problem.
+5. If you have a STACK DUMP decode it:
 
-if you have a stack dump decode it:
 https://github.com/esp8266/Arduino/blob/master/doc/Troubleshooting/stack_dump.rst
 
-for better debug messages:
-https://github.com/esp8266/Arduino/blob/master/doc/Troubleshooting/debugging.rst
+6. Include a Minimal Complete Reproducible Example sketch that shows your issue. Do not include your entire project, or a huge piece of code.
+7. Include debug messages:
+
+https://github.com/esp8266/Arduino/blob/master/doc/Troubleshooting/debugging.rst 

```

Comment:
How about using a readthedocs link here?

Reply:
Agreed, same for the stack dump.